libmcount/pyhook.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Python binding for mcount entry and exit
+ *
+ * Copyright (C) 2017, LG Electronics, Honggyu Kim <hong.gyu.kim@lge.com>
+ *
+ * Released under the GPL v2.
+ */
+#include <python2.7/Python.h>
```

Comment:
It supports only python2.7 as of now. python3 has a different APIs so it needs to be supported later on.